conf/initd/webvirtmgr-novnc-redhat

```diff
@@ -32,6 +37,7 @@ start() {
     [ -x $exec ] || exit 5
 
     echo -n $"Starting $prog: "
+    export PYTHONPATH=$PYTHONPATH:/usr/lib/python2.6/site-packages/Django-1.5.6-py2.6.egg
```

Comment:
This rpm works with EPEL package without any additional extra packages like pip. The versions are the same required in requirements.txt, I tested with version 1.5.6 and it works great. Anyway you fell free to add comment.